src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -157,6 +165,60 @@ const reducer=(state:IState,action:Action):IState=>{
             ...state,
             isDragEnter:action.value,
         }
+    case LOADING_FADEOUT:
+        return {
+            ...state,
+            loading:{
+                ...state.loading,
+                opacity:0,
+                position:true,
+            },
+        }
+    case LOADING_RESTORE:
+        return {
+            ...state,
+            loading:{
+                ...state.loading,
+                opacity:1,
+                position:false,
+            },
+        }
+    case ISSUCCESS_FADEOUT:
+        return {
+            ...state,
+            isSuccess:{
+                ...state.isSuccess,
+                opacity:0,
+                position:true,
+            },
+        }
+    case ISSUCCESS_RESTORE:
+        return {
+            ...state,
+            isSuccess:{
+                ...state.isSuccess,
+                opacity:1,
+                position:false,
+            },
+        }
+    case ISFAILURE_FADEOUT:
+        return {
+            ...state,
+            isFailure:{
+                ...state.isFailure,
+                opacity:0,
+                position:true,
+            },
+        }
+    case ISFAILURE_RESTORE:
+        return {
+            ...state,
+            isFailure:{
+                ...state.isFailure,
+                opacity:1,
+                position:false,
+            },
+        }
```

Comment:
Yes this is much better! I have a better understanding of why this action is taking place!